webdriver/tests/bidi/__init__.py

```diff
@@ -81,13 +81,7 @@ async def create_console_api_message(bidi_session, context, text):
     return text
 
 
-async def get_device_pixel_ratio(bidi_session, context):
-    """Get the DPR of the context.
-
-    :param bidi_session: BiDiSession
-    :param context: Browsing context ID
-    :returns: (float) devicePixelRatio.
-    """
```

Comment:
Why did you remove the documentation. It's still helpful because of the extra info.

Reply:
It adds no value.

- bidi_session is the name of the variable, just being repeated
- the return is the name of the function

Arguable the only piece of information the documentation provides is "context" = "browsing context id". I would have renamed the variable to browsing_context_id but for the sake of consistency (we use "context" everywhere) I did not.

Are you sure you want to add the docs back?

Reply:
Yeah you are right. We can leave it out.